packages/orbit-components/src/Select/index.tsx

```diff
@@ -167,11 +169,11 @@ const Select = React.forwardRef<HTMLSelectElement, Props>((props, ref) => {
             onBlur={onBlur}
             onChange={onChange}
             tabIndex={tabIndex ? Number(tabIndex) : undefined}
-            id={selectId}
             required={required}
             ref={ref}
             aria-describedby={shown ? `${selectId}-feedback` : undefined}
             aria-invalid={error ? true : undefined}
+            aria-label={ariaLabel as string}
```

Comment:
What happens if one does not pass `ariaLabel` prop? Does it apply `undefined` on the DOM? Or nothing at all?

Reply:
If `ariaLabel` is not passed, this attribute is `undefined`.